### PR TITLE
Climatic prepare

### LINKED ISSUE #4572: Small change in Climatic > Prepare dialogues
There are 6 dialogues in the Climatic > Prepare menu.
What is good about them all is that most fields are automatically completed.  That's the climatic "system".

A small edit would be useful in Transform, Climatic Summaries and Extremes.

In these dialogues the system cannot complete the element, because it could be one of many - in general.
But could it be the control with the initial focus, i.e. it should be in turquoise.  This is the case for the Spells dialogue already.

### PR DESCRIPTION
**Pull request by Muthenya**
@africanmathsinitiative/developers this is ready for review.
Fixes #4572

### CODE REVIEW

**Comment by Patowhiz** [comment]
Reviewing this

File: instat/dlgClimaticSummary.vb [hunk 1]
```diff
@@ -240,11 +241,11 @@ Public Class dlgClimaticSummary
...
-    Private Sub ReceiverFocus()
-        If rdoAnnual.Checked Then
-            ucrReceiverStation.SetMeAsReceiver()
-        End If
-    End Sub
+    'Private Sub ReceiverFocus()
+    '    If rdoAnnual.Checked Then
+    '        ucrReceiverStation.SetMeAsReceiver()
+    '    End If
+    'End Sub
```
Comment (dannyparsons) [inline instat/dlgClimaticSummary.vb]: These can be removed if not needed anymore

File: instat/dlgClimaticSummary.vb [hunk 2]
```diff
@@ -110,6 +102,15 @@ Public Class dlgClimaticSummary
...
         ucrReceiverDOY.strSelectorHeading = "Day Variables"
 
+        ' summary
+        ucrReceiverElement.SetParameter(New RParameter("columns_to_summarise", 0))
+        ucrReceiverElement.SetParameterIsString()
+        ucrReceiverElement.strSelectorHeading = "Variables"
+        ucrReceiverElement.Selector = ucrSelectorVariable
+        ucrReceiverElement.SetIncludedDataTypes({"numeric"})
+        ucrReceiverElement.bAutoFill = True
+        ucrReceiverElement.bWithQuotes = False
```
Comment (dannyparsons) [inline instat/dlgClimaticSummary.vb]: Did you need to add `bWithQuotes = False`?

**Comment by Patowhiz** [comment]
This works fine on my end. The element receiver for the dialogs in the climatic prepare menu gets the focus by default now.
@dannyparsons this is ready for you.

File: instat/dlgClimaticSummary.vb (hunk 2 above)
Comment (Muthenya) [reply instat/dlgClimaticSummary.vb]: This wasn't necessary.

**Pull request merged**